common_benchmark_suite/openxla/benchmark/models/pt/bert/bert_model.py

```diff
@@ -71,6 +77,8 @@ def create_model(batch_size: int = 1,
                  seq_len: int = 384,
                  data_type: str = "fp32",
                  model_name: str = "bert-large-uncased",
+                 import_on_gpu: bool = False,
+                 import_with_fx: bool = True,
```

Comment:
The method comment below also needs to be updated.

Reply:
Updated